src/client/actions.py

```diff
@@ -130,19 +130,18 @@ class Parameters(RuleAction.Parameters):
 
 
 class ChangeOverlap(RuleAction, spec_value=RuleType.CHANGE_OVERLAP):
-    """Increase the overlap of the set of tasks (or tasks, if the option is used "smart mixing")
+    """Increases the overlap of a task.
 
-    You can use this rule only with collectors.UsersAssessment and collectors.AssignmentsAssessment.
+    You can use this rule only with [UsersAssessment](https://toloka.ai/en/docs/toloka-kit/reference/toloka.client.collectors.UsersAssessment) and [AssignmentsAssessment](https://toloka.ai/en/docs/toloka-kit/reference/toloka.client.collectors.AssignmentsAssessment) collectors.
 
     Attributes:
-        parameters.delta: The number by which you want to increase the overlap of the task set
-            (or the task if the option is used "smart mixing").
-        parameters.open_pool: Changing the pool status:
-            * True - Open the pool after changing if it is closed.
-            * False - Do not open the pool after the change if it is closed.
+        parameters.delta: An overlap increment.
+        parameters.open_pool:
+            * `True` — Open the pool after changing the overlap value.
+            * `False` — Don't reopen the pool if it is closed.
 
     Example:
-        How to increase task overlap when you reject assignment in delayed mode.
+        The example shows how to increase task overlap when you reject assignments manually.
```

Comment:
I believe that "manually" is needed to be clarified: is it about just web UI or API also (in case of automatically handled post-acceptance pool, it is not "manual" strictly speaking). Let's clarify that this includes API calls.